en/assembly/quick_start_pixhawk4.md

```diff
@@ -61,30 +61,36 @@ M5 | I/O PWM OUT 5: connect signal wire to ESC of motor 5 here
 M6 | I/O PWM OUT 6: connect signal wire to ESC of motor 6 here
 M7 | I/O PWM OUT 7: connect signal wire to ESC of motor 7 here
 M8 | I/O PWM OUT 8: connect signal wire to ESC of motor 8 here
-FMU PWM-IN | See note below for connection to Pixhawk 4
-FMU PWM-OUT | If FMU PWM-IN is connected to Pixhawk 4, connect signal wires to ESC or signal, +, - wires to servos here
-CAP&ADC-OUT | connect to CAP & ADC IN port of Pixhawk 4
+FMU PWM-IN | See note below for connection to *Pixhawk 4*
+FMU PWM-OUT | If FMU PWM-IN is connected to *Pixhawk 4*, connect signal wires to ESC or signal, +, - wires to servos here
+CAP&ADC-OUT | connect to CAP & ADC IN port of *Pixhawk 4*
 CAP&ADC-IN | CAP&ADC input
 B+ | connect to ESC B+ to power the ESC
 GND | connect to ESC Ground
-PWR1 | 5v output 3A, connect to Pixhawk 4 POWER 1 
-PWR2 | 5v output 3A, connect to Pixhawk 4 POWER 2 
+PWR1 | 5v output 3A, connect to *Pixhawk 4* POWER 1 

```

Comment:
Markup ports exactly as shown in images (in bold). So this should be **POWER1**.

Reply:
Also, 5V not 5v in SI units.